releng/es.parent/pom.xml

```diff
@@ -35,7 +35,7 @@
 
 	<properties>
 		<elexis.target>2022-09-java17</elexis.target>
-
+		<elexis.branch>master</elexis.branch>
```

Comment:
No, don't do this! I removed it deliberately, s.t. the build run is not valid without passing the resp. variable!